scriptworker/cot/verify.py

```diff
@@ -1484,6 +1485,22 @@ def compare_jsone_task_definition(parent_link, compare_definition):
     return await verify_scriptworker_task(chain, obj)
 
 
+async def verify_bouncer_task(chain, obj):
```

Comment:
This follows the model, and is good. @Callek skipped it and is defining his task type as `scriptworker`, which also works because the `verify_INSTANCE_task` functions only call `verify_scriptworker_task`. I think we might want to move to that model at some point. Either way works for now.